lib/Page.js

```diff
@@ -84,6 +130,7 @@ Page.prototype.generate = function (builtFiles) {
       userDefinedVariablesMap: this.userDefinedVariablesMap,
       rootPath: this.rootPath,
     })
+      .then(result => this.collectFrontMatter(result))
```

Comment:
Nope, it should not. The front matter is removed during the page generation process.

Reply:
Does a static include result in a page having front matter?

Reply:
Yes. It'll inherit the frontmatter from the first static include if the page itself doesn't have its own front matter.

Reply:
@damithc Is that expected behaviour?

Reply:
> Yes. It'll inherit the frontmatter from the first static include if the page itself doesn't have its own front matter.

Can give an example where this is useful? My immediate thought is not to treat the _first_ include differently from any other includes.

Reply:
One useful scenario is when you want to include front matter from another file. I've updated the user guide with how it can be used here: [here[(https://deploy-preview-175--markbind.netlify.com/userguide/contentauthoring)

The changes to `site.json` are also mentioned [here](https://deploy-preview-175--markbind.netlify.com/userguide/siteconfiguration)